src/ext/java/org/opentripplanner/ext/traveltime/PostTransitSkipEdgeStrategy.java

```diff
@@ -0,0 +1,43 @@
+package org.opentripplanner.ext.traveltime;
+
+import java.time.Duration;
+import java.time.Instant;
+import org.opentripplanner.astar.spi.SkipEdgeStrategy;
+import org.opentripplanner.street.model.edge.Edge;
+import org.opentripplanner.street.search.state.State;
+
+public class PostTransitSkipEdgeStrategy implements SkipEdgeStrategy<State, Edge> {
+
+  private final long maxDurationSeconds;
+  private final long departureTime;
+  private final boolean arriveBy;
+
+  public PostTransitSkipEdgeStrategy(
+    Duration maxEgressTime,
```

Comment:
You probably want to rename the variable. It's not just egress.

Reply:
Technically it is either egress or direct, but now we have the same limit for both. Se might want to actually pass in both and use this on `TravelTimeStateData` and the direct limit on the regular `StateData`